### PR TITLE
filter graph by properties

### LINKED ISSUE #83: create property filters
Allow the user to filter the graph by property ranges.

e.g. add to the side panel as a separate section

### PR DESCRIPTION
**Pull request by sgratzl**
closes #83 

Screenshot from 2019-09-09 13-18-32

notes
 * I had to update `@material-ui` dependency. Thus, please run `yarn install`

### CODE REVIEW

File: src/datasets/precise/index.js [hunk 1]
```diff
@@ -18,13 +18,30 @@ export default {
         },
         formation_energy: {
             label: "Formation energy",
+            filterAble: true,
```
Comment (waxlamp) [inline src/datasets/precise/index.js]: Change this to be `filterable`.

In the future, we can replace this with simply knowing that a variable is continuous, right?

**Comment by sgratzl** [comment]
feedback integrated

**Comment by waxlamp** [comment]
Doesn't look like the slider debounce is working yet.

The size scale still seems to change after manipulating a slider.

Before:
before

After:
after

**Comment by sgratzl** [comment]
done

**Pull request merged**